Configuration/Test/test.py

```diff
@@ -0,0 +1,398 @@
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_020.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_021.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_022.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_023.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_024.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_025.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_100.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_101.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_102.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_103.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_104.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_105.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_120.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_121.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_122.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_123.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_124.xml',
+        'Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_125.xml',
+        'Geometry/VeryForwardData/data/RP_Device.xml',
+        'Geometry/VeryForwardData/data/RP_Vertical_Device.xml',
+        'Geometry/VeryForwardData/data/RP_Horizontal_Device.xml',
+        'Geometry/VeryForwardData/data/RP_220_Right_Station.xml',
+        'Geometry/VeryForwardData/data/RP_220_Left_Station.xml',
+        'Geometry/VeryForwardData/data/RP_147_Right_Station.xml',
+        'Geometry/VeryForwardData/data/RP_147_Left_Station.xml',
+        'Geometry/VeryForwardData/data/RP_Stations_Assembly.xml',
+        'Geometry/VeryForwardData/data/RP_Sensitive_Dets.xml',
+        'Geometry/VeryForwardData/data/RP_Cuts_Per_Region.xml',
+        'Geometry/VeryForwardData/data/RP_Param_Beam_Region.xml')
+
+process.XMLIdealGeometryESSource = cms.ESSource("XMLIdealGeometryESSource",
+    geomXMLFiles = totemGeomXMLFiles,
+    rootNodeName = cms.string('cms:CMSE')
+)
+
+# position of RPs
+process.XMLIdealGeometryESSource.geomXMLFiles.append("Geometry/VeryForwardData/data/2016_ctpps_15sigma_margin0/RP_Dist_Beam_Cent.xml")
+
+# extended geometries
+process.TotemRPGeometryESModule = cms.ESProducer("TotemRPGeometryESModule",
+    verbosity = cms.untracked.uint32(0)
+)
+
+# # Magnetic Field, by default we have 3.8T
+process.load("Configuration.StandardSequences.MagneticField_cff")
```

Comment:
Few screens below there is another definition of Magnetic Field. Why not to use only this one ?